test/integration/controller/jobset_controller_test.go

```diff
@@ -120,6 +119,11 @@ var _ = ginkgo.Describe("JobSet validation", func() {
 					gomega.Expect(k8sClient.Update(ctx, js)).Should(gomega.Succeed())
 				}
 			}
+
+			// Let's delete the jobset
+			gomega.Expect(k8sClient.Delete(ctx, js)).Should(gomega.Succeed())
```

Comment:
Shouldn't the `util.DeleteNamespace()` in `AfterEach()` handle this already? I see it deletes all jobsets.

Reply:
good call. updated.